style.css

```diff
@@ -78,7 +78,7 @@ h1,h2,h3,h4 {
 	
 	text-align:center;
 	padding: 0 5px;
-	font-family: 'Scream Real';
+	font-family: 'Avant Garde,Avantgarde,Century Gothic,CenturyGothic,AppleGothic,sans-serif; ';
```

Comment:
You need a link for these fonts on the each html page (or @include) because these fonts are not available for all Operating Systems/Browsers. Causing these fonts to not show up for everyone.